x-pack/plugin/rank-rrf/src/main/java/org/elasticsearch/xpack/rank/rrf/RRFRetrieverBuilder.java

```diff
@@ -342,4 +331,28 @@ public boolean doEquals(Object o) {
     public int doHashCode() {
         return Objects.hash(super.doHashCode(), fields, query, rankConstant, Arrays.hashCode(weights));
     }
+
+    /**
```

Comment:
nit: This is really a matter of style. Both the methods you extract here are beautifully named! Thus, what do the javadoc comments add? I personally prefer not to add comments if the comments do not add anything. No need to change anything here, just a first and last comment on comments :-)